### ISSUE TITLE
new list of malwares.

### ISSUE THREAD

**Issue by Axe-Usa** [ytisf/theZoo#85]

**Comment by SerialCrack3r** [comment]
Hello!

Can you name them? Are they classed in categories? How were they identified
as malwares?

**Comment by 912d** [comment]
@SerialCrack3r I do cannot name them, as I have obtained them a long time ago.
Here is download link for file "vir.7z". **Dangerous, don't run any**!
https://drive.google.com/open?id=186okY4jg1--lvw8n_qceornKhf7fdNt5

BTW, the most interesting malware is in virustotal.com database

**Comment by SerialCrack3r** [comment]
Yeah iuse virustotal but I don't think they let anyone download the samples
submitted. For that i just use malshare. It's not that bag and it even has
an API for automation... Anyway, thank you for sharing your samples. I'm
always happy to find new samples to study. Besides, it's even funnier to
try identify them..!

Have a nice day!

**Comment by 4llan** [comment]
Hey @jacekmusial, 

Could you, please, upload another mirror? These links are dead.
I'm very interested in your collection.

Thank you very much
